src/data/scripts/hullmods/VayraDamagedAmmo.java

```diff
@@ -5,18 +5,36 @@
 import com.fs.starfarer.api.combat.ShipAPI.HullSize;
 import com.fs.starfarer.api.combat.WeaponAPI.WeaponType;
 import com.fs.starfarer.api.impl.hullmods.CompromisedStructure;
+import data.scripts.util.MiscUtils;
+import data.util.LoggerLogLevel;
 
 public class VayraDamagedAmmo extends BaseHullMod {
+    private static final String LOGTAG = "VayraDamagedAmmo";
+    public static boolean DISABLE_FOR_PLAYER = false;
+    public static boolean DISABLE_FOR_ENEMY = false;
 
     @Override
     public void applyEffectsBeforeShipCreation(HullSize hullSize, MutableShipStatsAPI stats, String id) {
+        // Do nothing if it's been disabled
+        if (stats.getFleetMember() != null) {
+            int owner = stats.getFleetMember().getOwner();
+            if (owner == MiscUtils.OWNER_ENEMY && DISABLE_FOR_ENEMY) return;
+            if (owner == MiscUtils.OWNER_PLAYER && DISABLE_FOR_PLAYER) return;
```

Comment:
hate to say it, but the order of evaluating these should be reversed; check the constant first, and if it short-circuits there's no need to check who the actual owner is.

Reply:
done. resolving.